### ISSUE TITLE
Extremely poor contrast 

### ISSUE THREAD

**Issue by avivace** [indico/newdle#420]
The current color palette (e.g. the white text on bright pink gradient background) exhibits an extremely low score of color contrast

![image](https://user-images.githubusercontent.com/14352721/185099421-9e0434b3-b739-4e27-8cf0-bf076a076595.png)

See a tester: https://coolors.co/contrast-checker/fffefe-ff8aa1

We should try to make the website more accessible and readable by choosing more sensible color combinations.. see: https://en.wikipedia.org/wiki/Web_Content_Accessibility_Guidelines

Other examples of combinations with very poor contrast:

![image](https://user-images.githubusercontent.com/14352721/185099635-c0cc1d73-7cfe-4626-a248-b788b1ef0c32.png)

![image](https://user-images.githubusercontent.com/14352721/185099853-e84dd80e-368b-4925-ba82-a444831a4637.png)

**Comment by ThiefMaster** [comment]
Are there any CSS media queries indicating the client wants high contrast? The last time I checked there was only a Edge-only one which we of course don't want to use. Otherwise that'd be ideal, because:

- The footer text is meant to be inconspicuous so I'd say the "poor" contrast is intentional...  
- For the second one, using black text there looks worse IMHO

A11y is important, but should it make the UI look worse (or make "clutter" more visible) for others by doing so? I'd say not if it can be avoided!

**Comment by ThiefMaster** [comment]
Oh, there's actually this one: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-contrast

Maybe we could switch to higher contrast colors using that query...

**Comment by avivace** [comment]
> Are there any CSS media queries indicating the client wants high contrast? The last time I checked there was only a Edge-only one which we of course don't want to use. Otherwise that'd be ideal, because:
> 
>     * The footer text is meant to be inconspicuous so I'd say the "poor" contrast is intentional...
> 
>     * For the second one, using black text there looks worse IMHO
> 
> 
> A11y is important, but should it make the UI look worse (or make "clutter" more visible) for others by doing so? I'd say not if it can be avoided!

@ThiefMaster Sorry for my super delayed replies... The idea here is not to provide an "high contrast" mode (which can be useful and I agree too, as separate issue), but at least to reach acceptable values of contrast in the standard theme.

The [W3C’s Web Accessibility Initiative](https://www.w3.org/WAI/) provides a standard, which in the last version [WCAG 2.1](https://www.w3.org/TR/WCAG21/) has 4.5:1 as contrast in the "Success Criterion 1.4.3 Contrast".

Newdle is currently getting less than 2.13:1 in various UI elements, including the footer and the header text (Login):

![image](https://user-images.githubusercontent.com/14352721/199454769-5656caee-4060-4119-8190-f9a8109c33d3.png)

and less than 2.06:1 in the flashy-bright pink text over white (or the inverse). For large text the required ration is 3:1.

**Comment by ThiefMaster** [comment]
> I'm happy to try and iterate on some theme changes (while trying to keep the current "accents"), what are your opinions here?

Sure, that sounds good!
